### PR TITLE
Update getting-started.md

### PR DESCRIPTION
**Pull request by xfq**
Update URIs of developer tools. Some of them referred to an older version of developer tools before.

### CODE REVIEW

**Comment by arthurvr** [comment]
Hi @xfq! Thanks for all these PRs. In the future, can you try to format your commits [as specified by the style guide](http://contribute.jquery.org/commits-and-pull-requests/#commit-guidelines)? Thanks!

**Comment by xfq** [comment]
OK. Thanks for your attention.

BTW, what is the "Component" of chapters/articles learn.jquery.com? Just "learn.jquery.com"? Or the chapter name? Or title of the article? Or the file name? Or...?

**Comment by arthurvr** [comment]
@xfq see the commit log. The component mostly points to a specific section or article.

**Comment by xfq** [comment]
I see. Thank you.

File: page/javascript-101/getting-started.md [hunk 1]
```diff
@@ -76,8 +76,8 @@ One of JavaScript's greatest strengths is its simplicity. It can be written and
 
 Commonly referred to as "developer tools," many browsers ship with built-in features that provide better insight into JavaScript and jQuery while they run in the browser. Although they aren't required, you may find developer tools helpful when it comes to debugging errors in your code. Check out these browsers' developer tools:
 
-- [Apple Safari](https://developer.apple.com/technologies/safari/developer-tools.html)
-- [Google Chrome Developer Tools](https://developers.google.com/chrome-developer-tools/)
-- [Microsoft Internet Explorer](http://msdn.microsoft.com/en-us/library/ie/gg589507.aspx)
-- [Mozilla Firefox Developer Tools](https://developer.mozilla.org/en-US/docs/Tools)
+- [Apple Safari](https://developer.apple.com/safari/tools/)
+- [Google Chrome Developer Tools](https://developer.chrome.com/devtools)
+- [Microsoft Internet Explorer](http://msdn.microsoft.com/library/ie/bg182326(v=vs.85))
```
Comment (arthurvr) [inline page/javascript-101/getting-started.md]: Markdown counts that first closing bracket as the closing bracket for the link, so this results in:

screenshot 2015-01-21 06 52 02

**Comment by xfq** [comment]
I don't see any problem in [StackEdit](https://stackedit.io/) and [Markable](http://markable.in/). Anyway, I have updated the links. Thank you.

**Comment by xfq** [comment]
Because I'm not sure what `(v=vs.85)` means.

BTW, the [CommomMark Spec](http://spec.commonmark.org/0.16/#example-390) says that "one level of balanced parentheses is allowed without escaping" in the link destination.

**Pull request closed without merging**